doc_src/gear_developers/limitation.md

```diff
@@ -0,0 +1,12 @@
+# Limitation
+
+There are certain limitations in Antiktyera.
```

Comment:
```suggestion
There are certain limitations in Antikythera.
```

Reply:
I fixed in fa2d1e2e1b7a15428bbdf2bb4383356b46f4f95c